web/templates/user/index.html.eex

```diff
@@ -1,30 +1,9 @@
 <h2>Users</h2>
 
-<table class="table">
-  <thead>
-    <tr>
-      <th>Name</th>
-      <th>Email</th>
-      <th></th>
-      <th></th>
-    </tr>
-  </thead>
-  <tbody>
-<%= for user <- @users do %>
-    <tr>
-      <td><%= user.name %></td>
-      <td><%= user.email %></td>
-      <td><%= active_text(user) %></td>
-
-      <td class="text-right">
-        <%= link "Show", to: user_path(@conn, :show, user), class: "btn btn-default btn-xs" %>
-        <%= link "Edit", to: user_path(@conn, :edit, user), class: "btn btn-default btn-xs" %>
-        <%= link "Delete", to: user_path(@conn, :delete, user), method: :delete, data: [confirm: "Are you sure?"], class: "btn btn-danger btn-xs" %>
-      </td>
-    </tr>
+<div class="container">
+<%= for {user, changeset} <- @user_changesets do %>
+    <%= render "form.html", changeset: changeset, action: user_path(@conn, :update, user), conn: @conn %>
```

Comment:
@mbramson @Serneum once the changeset is available, we can then have a form/per user listed. I reused the form that was there to keep it simple but we could create another (with mo' better styling) to get the effect that's desired.